azext_iot/central/_help.py

```diff
@@ -512,6 +562,201 @@ def _load_central_roles_help():
     """
 
 
+def _load_central_organizations_help():
+    helps[
+        "iot central organization"
```

Comment:
Can `organization` be shortened to `org` for this command group, or is it important to spell it out?

Reply:
we prefer to be consistent to REST api naming